examples/angular/unit-testing/src/app/interceptor/mock-api.interceptor.ts

```
@@ -0,0 +1,64 @@
+        ) } else {
+          return respondWith(
+            200,
+            JSON.parse(
+              sessionStorage.getItem('unit-testing-tasks') || '[]',
+            ).concat([`CallNumber ${callNumber}`]),
+          )
+        }
+      case 'POST':
+        const tasks = JSON.parse(
+          sessionStorage.getItem('unit-testing-tasks') || '[]',
+        )
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)